public/app/features/explore/RichHistory/RichHistory.tsx

```diff
@@ -24,8 +24,8 @@ export enum Tabs {
 }
 
 export const sortOrderOptions = [
-  { label: 'Time ascending', value: SortOrder.Ascending },
```

Comment:
Based on suggestion at UX feedback session, using Newest first and Oldest first is more clear.